jetstream/resource_kv_bucket.go

```diff
@@ -142,25 +153,26 @@ func resourceKVBucketCreate(d *schema.ResourceData, m any) error {
 		return fmt.Errorf("bucket %s already exist", name)
 	}
 
-	js, err := nc.JetStream()
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	js, err := jetstream.New(nc)
 	if err != nil {
 		return err
 	}
 
-	_, err = js.CreateKeyValue(&nats.KeyValueConfig{
-		Bucket:       name,
-		Description:  descrption,
-		MaxValueSize: int32(maxV),
-		History:      uint8(history),
-		TTL:          time.Duration(ttl) * time.Second,
-		MaxBytes:     int64(maxB),
-		Storage:      nats.FileStorage,
-		Replicas:     replicas,
-		Placement:    placement,
+	js.CreateKeyValue(ctx, jetstream.KeyValueConfig{
```

Comment:
we should check the error here and replace the `mgr.IsKnownStream()` above with a load bucket from the `jetstream` package to check - make the whole thing use the `jetstream` package if possible